<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>F_Lot</class>
 <widget class="QDialog" name="F_Lot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Formulaire Lot</string>
  </property>
  <widget class="QComboBox" name="cbChoixArticle">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Choix de l'article:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Numero de lot Article:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leNumLotArticle">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Veuillez selectionner un article auquel sera
 ajouter le lot et préciser son numéro de lot 
ainsi que sa masse totale.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btValiderFormLot">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btAnnulerFormLot">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leMasseTotaleLot">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Masse Totale du lot :</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
